be/src/exprs/hll_function.cpp

```diff
@@ -32,16 +32,16 @@ StringVal HllFunctions::hll_hash(FunctionContext* ctx, const StringVal& input) {
     const int HLL_SINGLE_VALUE_SIZE = 10;
```

Comment:
add a comment to the number `10`? (1+1+8)?

Reply:
OK